site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputEmailFeedback" class="invalid-feedback">
+                  Please provide a valid email.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="selectCountry" id="selectCountryLabel" class="form-label is-required">Country</label>
+                <select class="form-select" id="selectCountry"
+                  aria-labelledby="selectCountryLabel selectCountryFeedback" required>
+                  <option selected disabled value="" aria-hidden="true">Select a country</option>
+                  <option value="1">France</option>
+                  <option value="2">Canada</option>
+                  <option value="3">USA</option>
+                  <option value="4">Australia</option>
+                </select>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="selectCountryFeedback" class="invalid-feedback">
+                  Please select a valid country.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputCompany" id="inputCompanyLabel" class="form-label is-required">Name of
+                  company</label>
+                <input type="text" class="form-control" id="inputCompany"
+                  aria-labelledby="inputCompanyLabel inputCompanyFeedback" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputCompanyFeedback" class="invalid-feedback">
+                  Please enter a valid company name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="selectBusiness" id="selectBusinessLabel" class="form-label is-required">Business
+                  type</label>
+                <select class="form-select" id="selectBusiness"
+                  aria-labelledby="selectBusinessLabel selectBusinessFeedback" required>
+                  <option selected disabled value="" aria-hidden="true">Select a type</option>
+                  <option value="1">Sole proprietorship</option>
+                  <option value="2">Partnership</option>
+                  <option value="3">Corporation</option>
+                  <option value="4">Limited liability company</option>
+                </select>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="selectBusinessFeedback" class="invalid-feedback">
+                  Please select a valid business type.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="textareaMessage" id="textareaMessageLabel" class="form-label is-required">Message</label>
```

Comment:
Missing information tooltip/popover next to the label

Reply:
The picture was indeed missing, fixed.